backend/core/quest.go

```diff
@@ -0,0 +1,11 @@
+package core
+
+// Quest represents a quest from the database
+type Quest struct {
```

Comment:
Hey, there have been some changes in the quests setup since creating the issue.

There are 2 quest types, `DailyQuests` and `MainQuests`. If you could expand this implementation to accommodate those, that'd be awesome. Sorry.

- [Daily Quests](https://github.com/keep-starknet-strange/art-peace/blob/3125497b8e9879fe63ddb7b96c2c1818489bc0c6/postgres/init.sql#L44)
- [Main Quests](https://github.com/keep-starknet-strange/art-peace/blob/3125497b8e9879fe63ddb7b96c2c1818489bc0c6/postgres/init.sql#L65)